### DISCUSSION TITLE
unable to use bra ket notation in Notion.

### DISCUSSION THREAD

**Discussion by SagarLboro** [KaTeX/KaTeX#2726] [Q&A]
I am unable to use `\ket` and `\bra` notations in Notion. Does anyone know if this is a common issue. The only workaround I found is `\vert \psi \rangle` and it's a lot of writing when instead I could just use `\ket \psi`.

**Comment by k4b7** [comment]
Support for `\bra` and `\ket` were added in 0.12.0.  Notion probably needs to upgrade their version of KaTeX.
